pkg/gotohelm/transpiler_test.go

```diff
@@ -101,16 +129,19 @@ func TestTranspile(t *testing.T) {
 		}
 	}
 
-	ctx := testutil.Context(t)
-	runner := NewGoRunner(td)
+	// Create and populate the test environment.
+	ctl := kubetest.NewEnv(t)
+	for _, obj := range seedObjects {
+		require.NoError(t, ctl.Create(context.Background(), obj))
+	}
 
-	go func() {
-		require.NoError(t, runner.Run(ctx))
-	}()
+	runner := NewGoRunner(t, td)
 
 	for _, pkg := range pkgs {
 		pkg := pkg
 		t.Run(pkg.Name, func(t *testing.T) {
+			ctx := testutil.Context(t)
```

Comment:
NIT: Why context is created within loop? Could there be single context that is capped by the go test framework?

Reply:
Hoisted.